.github/workflows/tests.yml

```diff
@@ -86,28 +96,15 @@ jobs:
           cache: "pip"
           cache-dependency-path: "pyproject.toml"
 
-      - name: Set up Node/yarn
-        uses: actions/setup-node@v1
-        with:
-          node-version: "16.x"
-
-      - name: Cache node_modules
-        uses: actions/cache@v2
-        with:
-          path: "node_modules"
-          key: |
-            ${{ runner.os }}-node-modules-${{ hashFiles('package.json') }}
-
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip wheel setuptools
           python -m pip install -e .[coverage]
-          yarn --frozen-lockfile
 
       # Build the docs
       - name: Build docs to store
         run: |
-          sphinx-build -b html docs/ docs/_build/html -W --keep-going
+          sphinx-build -b html docs/ docs/_build/html
```

Comment:
This removed the "turn warnings into errors" flag, but you added above a manual check for that. Is there a reason the manual check is better?

Reply:
Ah, nevermind, this is the audited docs, we don't need to error here again

Reply:
yep just removed it here because we're checking for warnings above already.

The manual check is because we **know** the kitchen sink has errors, and that is because we want to be able to demo what it looks like when an error is rendered into the actual page, since many of our users will do this